packages/modal/src/Modal/components/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { HTMLAttributes } from "react"
+import classnames from "classnames"
+import { Button, ButtonProps } from "@kaizen/button"
+import { OverrideClassName } from "@kaizen/component-base"
+import { useMediaQueries } from "@kaizen/responsive"
+import styles from "./ActionButton.scss"
+
+export interface ActionButtonProps
+  extends OverrideClassName<HTMLAttributes<HTMLDivElement>> {
+  isAlignedStart?: boolean
+}
+
+export const ActionButton: React.FC<ButtonProps & ActionButtonProps> = ({
```

Comment:
We aren't using FC anymore 😄

Reply:
I had a mental note to check what we are using again!  I knew it wasn't VFC lol but now I see it's neither!

Reply:
Simples! Haha